### PR TITLE
add delete history button in advanced->wallet info

### PR DESCRIPTION
**Pull request by javierbitpay**

### CODE REVIEW

**Comment by javierbitpay** [comment]
fixes #3358

**Comment by javierbitpay** [comment]
updated

**Comment by javierbitpay** [comment]
updated

File: src/js/controllers/preferencesInformation.js [hunk 1]
```diff
@@ -94,4 +93,16 @@ angular.module('copayApp.controllers').controller('preferencesInformation',
         });
       }, 100);
     };
+
+    this.clearTransactionHistory = function() {
+      storageService.removeTxHistory(c.walletId, function(err) {
+        if (err) return $log.err(err);
```
Comment (cmgustavo) [inline src/js/controllers/preferencesInformation.js]: It should be: 
``` 
$log.err(err); 
return;
```

File: src/js/controllers/index.js [hunk 2]
```diff
@@ -1005,6 +1005,12 @@ angular.module('copayApp.controllers').controller('indexController', function($r
   };
 
   // UX event handlers
```
Comment (cmgustavo) [inline src/js/controllers/index.js]: It 's not an UX event

**Comment by javierbitpay** [comment]
update

File: src/js/controllers/preferencesInformation.js [hunk 3]
```diff
@@ -94,4 +93,19 @@ angular.module('copayApp.controllers').controller('preferencesInformation',
         });
       }, 100);
     };
+
+    this.clearTransactionHistory = function() {
+      storageService.removeTxHistory(c.walletId, function(err) {
+        if (err) {
+          $log.err(err);
```
Comment (cmgustavo) [inline src/js/controllers/preferencesInformation.js]: $log.err doesn't exist. https://docs.angularjs.org/api/ng/service/$log

Comment (javierbitpay) [inline src/js/controllers/preferencesInformation.js]: fixed

**Comment by cmgustavo** [comment]
ACK

**Pull request merged**